lib/commons/color/get-background-color.js

```diff
@@ -53,40 +53,47 @@ function _getBackgroundColor(elm, bgElms, shadowOutlineEmMax) {
   }
 
   const elmStack = getBackgroundStack(elm);
+  const textRects = getVisibleChildTextRects(elm);
```

Comment:
I'm not sure about this. We're now calling getVisibleChildTextRects twice. Once here, and once inside `getBackgroundStack > getTextElementStack`. What if we passed these rects into getBackgroundStack instead so we don't need to make the call twice?

```suggestion
  const textRects = getVisibleChildTextRects(elm);
  const elmStack = getBackgroundStack(elm, textRects);
```

Reply:
Alternatively, memoize for getVisibleChildTextRects could work too.